config/config_file.go

```diff
@@ -1,109 +1,84 @@
 package config
 
 import (
+	"context"
 	"fmt"
+	"io/ioutil"
 	"os"
-	"os/user"
 	"path/filepath"
 
+	"go4.org/xdgdir"
 	"gopkg.in/ini.v1"
+	"zombiezen.com/go/log"
 )
 
-func configFilePath() (string, error) {
-	u, err := user.Current()
-
-	if err != nil {
-		return "", err
-	}
-
-	configDir := filepath.Join(u.HomeDir, ".config", "yb")
-	mkdirAsNeeded(configDir)
-	iniPath := filepath.Join(configDir, "settings.ini")
+const (
+	dirName      = "yb"
+	settingsName = "settings.ini"
+)
 
-	if !pathExists(iniPath) {
-		emptyFile, _ := os.Create(iniPath)
-		emptyFile.Close()
+func loadConfigFiles() (*ini.File, error) {
+	// Gather INI files in reverse order because earlier entries should override
+	// later entries.
+	searchPaths := xdgdir.Config.SearchPaths()
```

Comment:
After reading this comment I believed "earlier"/"later" referred to timestamp, which was confusing enough that I was compelled to dig into `xdgdir` and `ini` docs to unravel the mystery. Can we clarify this comment a bit to explain it upfront?
```suggestion
	// xdgdir.Config.SearchPaths returns config files in descending order of preference,
	// but ini.Load accepts them in ascending order of preference, so gather them in reverse.
	searchPaths := xdgdir.Config.SearchPaths()
```
(feel free to edit)